test/frontend/actions/buildLogActionsTest.js

```diff
@@ -37,7 +37,8 @@ describe("buildActions", () => {
     const buildLogsPromise = Promise.resolve(logs);
     const fetchStartedAction = { action: "fetchStarted" }
     const receivedAction = { action: "received" };
-    fetchBuildLogs.withArgs().returns(buildLogsPromise);
+    fetchBuildLogs.withArgs().onCall(0).returns(buildLogsPromise);
+    fetchBuildLogs.withArgs().onCall(1).returns([]);
```

Comment:
We may want to make this `.returns(Promise.resolve([]))`, right?

Reply:
👍